src/Configurations/test/Responses.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { waitFor } from '@testing-library/react';
+
+import * as components from '@folio/stripes-acq-components';
+
+import { mockKy, rest, server } from '../../test/net';
+import { ERROR_RESPONSE, renderConfigurations, url } from './setup';
+
+
+jest.mock('@folio/stripes-acq-components', () => ({
+  ...jest.requireActual('@folio/stripes-acq-components'),
+  useShowCallout: jest.fn(),
+}));
+
+jest.mock('react-virtualized-auto-sizer', () => ({ children }) => children({ width: 1920, height: 1080 }));
+
+jest.mock('@folio/stripes/core', () => ({
+  ...jest.requireActual('@folio/stripes/core'),
+  useOkapiKy: () => mockKy,
+  useStripes: () => ({
+    hasPerm: jest.fn().mockReturnValue(true),
+  }),
+  IfPermission: props => <>{props.children}</>,
+}));
+
+
+describe('EditorLayer', () => {
+  it('`useSingleQuery` fires showCalloutHook if any error', async () => {
```

Comment:
```suggestion
describe('Fetching single configuration', () => {
  it('shows error callout in editor, in case of server error', async () => {
```
This way we group the tests thematically. So we could add here the test for 'Fetching single configuration' in details pane, for example (which is right thing to do for this ticket).

And in the test description we get rid of implementation details like `useSingleQuery` and `showCalloutHook` (which is non-existent BTW)